.github/workflows/stale.yml

```
@@ -0,0 +1,40 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+    # Run once a day
+    - cron: "30 1 * * *"
+
+  push:
+    branches:
+      - stale
+
+jobs:
+  stale:
+    if: "github.repository == 'ESMValGroup/ESMValTool'"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          debug-only: true
+          days-before-issue-stale: 1080
+          days-before-pr-stale: 360
+          days-before-close: 60
+          ascending: true
+          operations-per-run: 1000
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 1080 days of inactivity.
+
+            If this issue is still important to you, please comment below to remove the stale label. Otherwise, this issue will be automatically closed in 60 days.
+          stale-pr-message: |
+            In order to maintain a backlog of relevant pull requests, we automatically label them as stale after 360 days of inactivity.
+
+            If this pull request is still important to you, please comment below to remove the stale label. Otherwise, this pull request will be automatically closed in 60 days. If this pull request only suffers from a lack of reviewers, please tag the @<span></span>ESMValGroup/technical-lead-development-team so they can help you find a suitable reviewer.
+          close-issue-message: |
+            This stale issue has been automatically closed due to a lack of community activity.
+
+            If you still care about this issue, please re-open it. If you do not have sufficient permissions for this, tag the @<span></span>ESMValGroup/technical-lead-development-team in a comment below including a short message that this issue is still relevant to you. They will re-open it on your behalf.
+          close-pr-message: |
+            This stale pull request has been automatically closed due to a lack of community activity.
+
+            If you still care about this pull request, please re-open it. If you do not have sufficient permissions for this, tag the @<span></span>ESMValGroup/technical-lead-development-team in a comment below including a short message that this pull request is still relevant to you. They will re-open it on your behalf.
```

```suggestion
            If you still care about this pull request, please re-open it. If you do not have sufficient permissions for this, tag the @<span></span>ESMValGroup/esmvaltool-coreteam in a comment below including a short message that this pull request is still relevant to you. They will re-open it on your behalf.
```
Same here, I don't think we need purely the tech lead team to reopen in the Tool.

---

I thought the core team was just the combined TLT and SLT but looking at the members you're right. TLT and SLT are a little less outdated than core team =D

---

Changed in https://github.com/ESMValGroup/ESMValTool/pull/4100/commits/408c9f15af79cd93f346419bd24986c0cc704661